### ISSUE TITLE
Error: Building package qt5-declarative:x64-windows-static failed with: BUILD_FAILED

### ISSUE THREAD

**Issue by kingofalpha** [microsoft/vcpkg#7102]
We recommend moving vcpkg to a short path such as 'C:\src\vcpkg' or using
  the subst command.
Call Stack (most recent call first):
  installed/x64-windows-static/share/qt5modularscripts/qt_modular_library.cmake:159 (qt_modular_fetch_library)
  ports/qt5-declarative/portfile.cmake:5 (qt_modular_library)
  scripts/ports.cmake:73 (include)

-- Using cached D:/vcpkg-master/downloads/qtdeclarative-everywhere-src-5.12.3.tar.xz
-- Using source at D:/vcpkg-master/buildtrees/qt5-declarative/src/5.12.3-93b4ba5375
-- Configuring x64-windows-static-rel
-- Configuring x64-windows-static-rel done
-- Configuring x64-windows-static-dbg
-- Configuring x64-windows-static-dbg done
-- Package makefiles-x64-windows-static-dbg
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:56 (message):
    Command failed: D:/vcpkg-master/downloads/tools/jom/jom-1.1.3/jom.exe;qmake_all
    Working Directory: D:/vcpkg-master/buildtrees/qt5-declarative/x64-windows-static-dbg
    Error code: 2
    See logs for more information:
      D:\vcpkg-master\buildtrees\qt5-declarative\package-makefiles-x64-windows-static-dbg-out.log
      D:\vcpkg-master\buildtrees\qt5-declarative\package-makefiles-x64-windows-static-dbg-err.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_build_qmake.cmake:48 (vcpkg_execute_required_process)
  scripts/cmake/vcpkg_build_qmake.cmake:63 (run_jom)
  installed/x64-windows-static/share/qt5modularscripts/qt_modular_library.cmake:56 (vcpkg_build_qmake)
  installed/x64-windows-static/share/qt5modularscripts/qt_modular_library.cmake:160 (qt_modular_build_library)
  ports/qt5-declarative/portfile.cmake:5 (qt_modular_library)
  scripts/ports.cmake:73 (include)

Error: Building package qt5-declarative:x64-windows-static failed with: BUILD_FAILED

**Comment by Voskrese** [comment]
**See logs for more information:**
```
D:\vcpkg-master\buildtrees\qt5-declarative\package-makefiles-x64-windows-static-dbg-out.log
D:\vcpkg-master\buildtrees\qt5-declarative\package-makefiles-x64-windows-static-dbg-err.log
```

**Comment by cenit** [comment]
duplicate of #6962?

**Comment by wangli28** [comment]
Thanks for your feedback! Closed as duplicate of #4560.
